packages/angular/src/table/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './table.module';
```

Comment:
I think we need a `package.json` for this folder as well (so it gets built as a separate module)